anet-dictionary.yml

```diff
@@ -202,6 +202,8 @@ fields:
     nextSteps: Next steps
     keyOutcomes: Key outcomes
     reportText: Key details
+    detailedView: Detailed View
+    compactView: Summary / Print
```

Comment:
These are not report fields, just buttons, so they don't belong here.
I'd say it is just fine to hard-code these into Show.js and CompactView.js.

Reply:
I also think it is fine, there were user feedback to change the previous hard-coded labels, and i thought might as well give admins a way to change when needed. If @VassilIordanov doesn't have a strong requirement, i can revert or put it somewhere else.